docs/candid/01-style-guide.md

```diff
@@ -2,3 +2,306 @@
 title: Candid Style Guide
 description: A style guide for writing Candid interfaces for Internet Computer canisters.
 ---
+
+## Candid
+
+Candid is a serialization format for the Internet Computer. It is used to define the interface of canisters, and to encode and decode messages that are sent to and from canisters.
+
+## Motivation
+
+Canisters are written in a variety of languages, as are the clients that interact with canisters, while Candid is designed to be language agnostic. This means that Candid interfaces should follow a consistent pattern that is not biased by the language used to write the canister interfaces it is describing.
```

Comment:
"biased by" is not a correct form, rephrasing would be good. Maybe "biased towards"? Although this is not exactly the meaning you want here.

In my view "it is describing" could be removed without loss of meaning.

Reply:
Chat GPT suggests this:

> “Canisters and their client interactions are written in various languages. However, Candid, which is language-agnostic, describes these interfaces. Therefore, Candid interfaces should maintain a consistent pattern, unbiased by the language used in the canister interfaces they describe.”

WDYT?